# MODEL1303140001: MODEL1303140001

## Installation

Download this repository, and install with distutils

`python setup.py install`

Or, install using pip

`pip install git+https://github.com/biomodels/MODEL1303140001.git`

To install a specific version (in this example, from the 2014-09-16 BioModels release)

`pip install git+https://github.com/biomodels/MODEL1303140001.git@20140916`


# Model Notes


Ray2013 - S.cerevisiae meiosis-specific metabolic network

Meiosis is a strongly concerved cell division program that generates haploid
gametes from a diploid parental cell. Successful meiosis is the fundamental
basis of sexual reproduction. Multiple lines of evidence suggest a tight link
between meiosis and metabolism. Here, yeast meiosis is studied to elucidate
the link between reproduction and metabolism. Network flux is obtained using
GLPK (GNU Linear Programming Kit) supported by the COBRA Toolbox for Matlab.

This model is described in the article:

[Characterization of the metabolic requirements in yeast
meiosis.](http://identifiers.org/pubmed/23675502)

Ray D, Ye P.

PLoS One. 2013 May 8;8(5):e63707.

Abstract:

The diploid yeast Saccharomyces cerevisiae undergoes mitosis in glucose-rich
medium but enters meiosis in acetate sporulation medium. The transition from
mitosis to meiosis involves a remarkable adaptation of the metabolic machinery
to the changing environment to meet new energy and biosynthesis requirements.
Biochemical studies indicate that five metabolic pathways are active at
different stages of sporulation: glutamate formation, tricarboxylic acid
cycle, glyoxylate cycle, gluconeogenesis, and glycogenolysis. A dynamic
synthesis of macromolecules, including nucleotides, amino acids, and lipids,
is also observed. However, the metabolic requirements of sporulating cells are
poorly understood. In this study, we apply flux balance analyses to uncover
optimal principles driving the operation of metabolic networks over the entire
period of sporulation. A meiosis-specific metabolic network is constructed,
and flux distribution is simulated using ten objective functions combined with
time-course expression-based reaction constraints. By systematically
evaluating the correlation between computational and experimental fluxes on
pathways and macromolecule syntheses, the metabolic requirements of cells are
determined: sporulation requires maximization of ATP production and
macromolecule syntheses in the early phase followed by maximization of
carbohydrate breakdown and minimization of ATP production in the middle and
late stages. Our computational models are validated by in silico deletion of
enzymes known to be essential for sporulation. Finally, the models are used to
predict novel metabolic genes required for sporulation. This study indicates
that yeast cells have distinct metabolic requirements at different phases of
meiosis, which may reflect regulation that realizes the optimal outcome of
sporulation. Our meiosis-specific network models provide a framework for an
in-depth understanding of the roles of enzymes and reactions, and may open new
avenues for engineering metabolic pathways to improve sporulation efficiency.

This model is hosted on [BioModels Database](http://www.ebi.ac.uk/biomodels/)
and identified by:
[MODEL1303140001](http://identifiers.org/biomodels.db/MODEL1303140001) .

To cite BioModels Database, please use: [BioModels Database: An enhanced,
curated and annotated resource for published quantitative kinetic
models](http://identifiers.org/pubmed/20587024) .

To the extent possible under law, all copyright and related or neighbouring
rights to this encoded model have been dedicated to the public domain
worldwide. Please refer to [CC0 Public Domain
Dedication](http://creativecommons.org/publicdomain/zero/1.0/) for more
information.


